hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/DeletingServiceMetrics.java

```diff
@@ -160,6 +166,18 @@ public void incrNumRenameEntriesPurged(long renameEntriesPurged) {
     this.numRenameEntriesPurged.incr(renameEntriesPurged);
   }
 
+  public TransactionInfo getLastAOSTransactionId() {
+    return TransactionInfo.valueOf(lastAOSPurgeTermId.value(), lastAOSPurgeTransactionId.value());
+  }
+
+  public synchronized void setLastAOSTransactionId(TransactionInfo transactionInfo) {
+    TransactionInfo previousTransactionInfo = getLastAOSTransactionId();
+    if (transactionInfo.compareTo(previousTransactionInfo) > 0) {
+      this.lastAOSPurgeTransactionId.set(transactionInfo.getTerm());
+      this.lastAOSPurgeTransactionId.set(transactionInfo.getTransactionIndex());
```

Comment:
this is probably wrong. See: https://github.com/jojochuang/ozone/pull/583#discussion_r2143770452

Reply:
done